fxgl-core/src/main/kotlin/com/almasb/fxgl/audio/AudioPlayer.kt

```diff
@@ -74,7 +57,6 @@ class AudioPlayer : EngineService() {
         if (!activeSounds.containsByIdentity(sound))
             activeSounds.add(sound)
 
-        sound.audio.setVolume(soundVolume.value)
```

Comment:
No need anymore, the global volumes are automatically mixed with the Media configured sound